consensus/istanbul/backend/engine.go

```diff
@@ -172,7 +172,8 @@ func (sb *backend) computeSignatureAddrs(header *types.Header) error {
 // given engine. Verifying the seal may be done optionally here, or explicitly
 // via the VerifySeal method.
 func (sb *backend) VerifyHeader(chain consensus.ChainReader, header *types.Header, seal bool) error {
-	return sb.verifyHeader(chain, header, nil)
+	parents := []*types.Header{chain.GetHeader(header.ParentHash, header.Number.Uint64())}
```

Comment:
According to [verifyCascadingFields()](https://github.com/klaytn/klaytn/blob/v1.8.4/consensus/istanbul/backend/engine.go#L206-L218), you should use `number-1` to find parent header.

```suggestion
        parents := []*types.Header{}
        number := header.Number.Uint64()
        if number > 0 {
                parents = append(parents, chain.GetHeader(header.ParentHash, number-1))
        }
```

Reply:
@blukat29 Thanks! great code :)